### PR TITLE
chore: adapt benchmarking improvement in pallet-move

### PR DESCRIPTION
**Pull request by neutrinoks**
- update Cargo.lock (changes in substrate-move and pallet-move)
- adapted pallet-move's default weight definition (`SubstrateWeight` to `WeightInfo`)

### CODE REVIEW

File: runtime/Cargo.toml [hunk 1]
```diff
@@ -55,7 +56,7 @@ pallet-move = { path = "../../pallet-move/pallet", default-features = false }
 pallet-template = { path = "../pallets/template", default-features = false }
 
 [build-dependencies]
-substrate-wasm-builder = { git = "https://github.com/paritytech/polkadot-sdk.git", branch = "release-polkadot-v1.4.0", optional = true }
+substrate-wasm-builder = { git = "https://github.com/paritytech/polkadot-sdk.git", branch="release-polkadot-v1.4.0" }
```
Comment (Rqnsom) [inline runtime/Cargo.toml]:
```suggestion
substrate-wasm-builder = { git = "https://github.com/paritytech/polkadot-sdk.git", branch = "release-polkadot-v1.4.0" }
```

File: runtime/Cargo.toml [hunk 2]
```diff
@@ -32,6 +32,7 @@ sp-consensus-aura = { git = "https://github.com/paritytech/polkadot-sdk.git", br
 sp-consensus-grandpa = { git = "https://github.com/paritytech/polkadot-sdk.git", branch = "release-polkadot-v1.4.0", default-features = false, features = ["serde"] }
 sp-core = { git = "https://github.com/paritytech/polkadot-sdk.git", branch = "release-polkadot-v1.4.0", default-features = false, features = ["serde"]}
 sp-inherents = { git = "https://github.com/paritytech/polkadot-sdk.git", branch = "release-polkadot-v1.4.0", default-features = false }
+sp-io = { git = "https://github.com/paritytech/polkadot-sdk.git", branch ="release-polkadot-v1.4.0", default-features = false }
```
Comment (Rqnsom) [inline runtime/Cargo.toml]:
```suggestion
sp-io = { git = "https://github.com/paritytech/polkadot-sdk.git", branch = "release-polkadot-v1.4.0", default-features = false }
```

**Review by Rqnsom — DISMISSED** [review]
LGTM

File: runtime/Cargo.toml (hunk 1 above)
Comment (neutrinoks) [reply runtime/Cargo.toml]: Done

File: runtime/Cargo.toml (hunk 2 above)
Comment (neutrinoks) [reply runtime/Cargo.toml]: Done

**Review by Rqnsom — APPROVED** [review]

**Pull request merged**